test/e2e/performanceprofile/functests/7_test_test/test_suite_test_test.go

```diff
@@ -0,0 +1,40 @@
+package pao_mustgather
```

Comment:
this new file is part of moving to v2?  do we need to align and add before suite functions ?  if so, do we apply them in our code?

Reply:
Remove